src/Checkbox/Checkbox.tsx

```diff
@@ -79,7 +80,6 @@ const VisualCheckbox: React.FunctionComponent<any> = styled.div(
     borderRadius: theme.radii.small,
     border: "solid 1px",
     position: "relative",
-    alignSelf: "center",
```

Comment:
Would changing this to `baseline` serve the same purpose as adding the `marginTop` below at line 136?

Reply:
@haideralsh not quite. Using `baseline` does align the bottom of the checkbox with the baseline of the first line of text, but the designers wanted the bottom of the checkbox slightly lower than the baseline of the first line of text. If you know a better way to do this I'm all ears though!

Reply:
Here's a link to the figma: https://www.figma.com/file/uLRNnoLgFamkSsy9zhAcPm/Capacity-Management?node-id=4283%3A157379

Reply:
Thanks for the explanations @sskirby. I see why you went with this approach now.